src/cuda/image_function_cuda.cu

```diff
@@ -239,6 +239,59 @@ namespace
         }
     }
 
+    __global__ void rotateCuda( const uint8_t * in, uint32_t rowSizeIn, uint32_t inHeight, float inXStart, float inYStart, 
+                                uint8_t * out, uint32_t rowSizeOut, uint32_t outHeight, 
+                                float cosAngle, float sinAngle)
```

Comment:
Move this code to previous line. We have a recommended line size as 150 characters so it should be fine.

Reply:
I moved `float cosAngle, float sinAngle)` to line above.